core/capabilities/ccip/oraclecreator/bootstrap.go

```diff
@@ -64,19 +80,32 @@ func (i *bootstrapOracleCreator) Create(_ uint32, config cctypes.OCR3ConfigWithM
 		return nil, fmt.Errorf("failed to get chain ID from selector: %w", err)
 	}
 
+	ctx := context.Background()
+	rmnHomeReader, err := i.getRmnHomeReader(ctx, config)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get RMNHome reader: %w", err)
+	}
+	if err = rmnHomeReader.Start(ctx); err != nil {
+		return nil, fmt.Errorf("failed to start RMNHome reader: %w", err)
+	}
+
 	destChainFamily := chaintype.EVM
 	destRelayID := types.NewRelayID(string(destChainFamily), fmt.Sprintf("%d", chainID))
 
-	// i.peerWrapper.PeerGroupFactory.NewPeerGroup()...
-	// get all config digest from rmn home in home chain
-	// for each config digest:
-	// i.bootstrapperLocators
-	// fetch from rmn home rmn peer ids
-	// figure out how to get oracle peer ids
-	// i.peerWrapper.PeerGroupFactory.NewPeerGroup()
-	//
-	// watch configs, close peer groups and open new ones when configs change
-	// _, _ := i.peerWrapper.PeerGroupFactory.NewPeerGroup()
+	oraclePeerIDs := make([]ragep2ptypes.PeerID, 0, len(config.Config.Nodes))
+	for _, n := range config.Config.Nodes {
+		oraclePeerIDs = append(oraclePeerIDs, n.P2pID)
+	}
+
+	pgd := newPeerGroupDialer(
+		i.lggr.Named("PeerGroupDialer"),
+		i.peerWrapper.PeerGroupFactory,
+		rmnHomeReader,
+		i.bootstrapperLocators,
+		oraclePeerIDs,
+		config.ConfigDigest,
+	)
+	pgd.Start()
```

Comment:
As discussed offline this is cleaner if called when the bootstrap `Start` is called by the launcher. My recommendation is to wrap this and the bootstrap and have a `Start` call on that start both this background service and the OCR bootstrapper.